### ISSUE TITLE
Get hyperparameter tuning results-- add additional realizations

### ISSUE THREAD

**Issue by anonml2020** [anonml2020/gn#5]
- [x] Set up algorithm to run over basic parameters for accuracy
- [x] Configure algorithm to record performance **and** accuracy
- [x] Once initial optimization of the code is finished, start a batched run of all benchmarks
- [x] Expand algorithm to change data parameters like cutoff distance

**Comment by anonml2020** [comment]
Do we want to do a grid search? Or find hyperparameter with Bayesian optimization?

@ckreisch @jdh4 @franciscovillaescusa

**Comment by anonml2020** [comment]
Best results were found on best combined trial was: loss = 0.08, time=11 seconds, batch=175, batch_per_epoch=14 (I run 20 epochs total), cutoff=20, hidden=270, l1=1e-4, l2=3.7e-7, msg_dimension=5.

Here are some plots:
![image](https://user-images.githubusercontent.com/61266943/84167893-cb16f980-aa44-11ea-9f52-ed1f3935dd90.png)
![image](https://user-images.githubusercontent.com/61266943/84167902-cd795380-aa44-11ea-93e6-58146e107040.png)
